tools/dgeni/src/test.ts

```diff
@@ -0,0 +1,22 @@
+// This file is required by karma.conf.js and loads recursively all the .spec and framework files
+
+import 'core-js/es7/reflect';
```

Comment:
I don't think we need this file for the tests either.

Reply:
Removed